xbmc/platform/linux/LinuxTimezone.cpp

```diff
@@ -153,6 +156,9 @@ void CLinuxTimezone::OnSettingChanged(std::shared_ptr<const CSetting> setting)
   const std::string &settingId = setting->GetId();
   if (settingId == CSettings::SETTING_LOCALE_TIMEZONE)
   {
+#ifdef HAS_DBUS
+    CSystemdUtils::SetTimezone(std::static_pointer_cast<const CSettingString>(setting)->GetValue());
```

Comment:
I'm not sure we always want to do that. On Desktop systems, users usually don't have the right to call `SetTimezone` on timedated, so you'll get error log spam. And even if they have the right, it's totally unexpected (to me at least) that changing the time zone in Kodi would change the system time zone.

Reply:
I totally agree. The PR was done completely with LibreELEC in mind

Reply:
I know that on LibreELEC, Kodi runs as root. Is that expected to change at some point or will you continue to have that?

Also, do we have sth like a "standalone/management" mode for boxes only running Kodi (like LibreELEC basically...) that we could use for enabling this feature?